### PR TITLE
Update CDI visibility FAT and run on EE10

### PR DESCRIPTION
**Pull request by Azquelt**
For #19867

### CODE REVIEW

**Comment by Azquelt** [comment]
#libby
#build

File: dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml]: is this intentionally unversioned?

Comment (Azquelt) [reply dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml]: I don't think so, though it is consistent with the other test archives for the warlibs test.

Should probably update all of them to have a version.

File: dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/sharedlib/crossInjectionLib2Jar/beans.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xmlns="http://xmlns.jcp.org/xml/ns/javaee" bean-discovery-mode="annotated"> </beans>
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/sharedlib/crossInjectionLib2Jar/beans.xml]: version=?

File: dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/sharedlib/crossInjectionWar/beans.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xmlns="http://xmlns.jcp.org/xml/ns/javaee" bean-discovery-mode="annotated"> </beans>
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/sharedlib/crossInjectionWar/beans.xml]: version=?

**Review by tevans78 — APPROVED** [review]
The beans.xml files should have a version attribute but this doesn't break anything ... so approving.

File: dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml (hunk 1 above)
Comment (Azquelt) [reply dev/com.ibm.ws.cdi.visibility_fat/fat/src/com/ibm/ws/cdi/visibility/tests/basic/warlibs/webinfLibJar/beans.xml]: Versions added to all beans.xmls

**Comment by Azquelt** [comment]
I've added versions to all beans.xmls

#libby
#build

**Pull request merged**